client/my-sites/stats/activity-log-tasklist/update.jsx

```diff
@@ -42,17 +41,15 @@ class ActivityLogTaskUpdate extends Component {
 	handleNameClick = () => this.props.goToPage( this.props.slug, this.props.type );
 
 	render() {
-		const {
-			translate,
-			name,
-			version,
-			type,
-			updateType,
-			disable,
-			linked,
-			slug,
-			siteSlug,
-		} = this.props;
+		const { translate, name, version, type, disable, linked, slug, siteSlug } = this.props;
+
+		let updateType = translate( 'Plugin update available' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **19** times:
`translate( 'Plugin update available', 'Plugin updates available' )` *ES Score: 16*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/activity-log-updates-listing)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).